articles/hdinsight/domain-joined/apache-domain-joined-configure-using-azure-adds.md

```diff
@@ -23,6 +23,10 @@ In this article, you learn how to configure a HDInsight cluster with ESP by usin
 
 Enabling Azure AD-DS is a prerequisite before you can create a HDInsight cluster with ESP. For more information, see [Enable Azure Active Directory Domain Services using the Azure portal](../../active-directory-domain-services/active-directory-ds-getting-started.md). 
 
+After enabling AAD-DS, all users and objects will start synchronizing from AAD to AAD-DS by default. The length of the sync operation depends on the number of objects in AAD. For hundreds of thousands of objects, it could take a few days.
+
+Customers can choose to sync only those groups that will need access to the HDInsight clusters. This option of syncing only certain groups is called *scoped synchronization*. See [Configure Scoped Synchronization from Azure AD to your managed domain](https://docs.microsoft.com/en-us/azure/active-directory-domain-services/active-directory-ds-scoped-synchronization) for instructions.
```

Comment:
Please remote "those" in the first sentence. Also remove "will" to stay in present tense.